db/seeds.rb

```diff
@@ -5,3 +5,23 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+User.create!([
+    {name: "Tom", photo: "https://unsplash.com/photos/F_-0BxGuVvo", bio: "Teacher from Mexico.", posts_counter: 4},
+    {name: "Lilly", photo: "https://unsplash.com/photos/F_-0BxGuVvo", bio: "Teacher from Poland.", posts_counter: nil}
+  ])
+  
+  Post.create!([
+  {author: User.first,title: "Hello2", text: "This is my second post", comments_counter: nil, likes_counter: nil },
+  {author: User.first, title: "Hello3", text: "This is my Third post", comments_counter: nil, likes_counter: nil },
+  {author: User.find(2),title: "Hello4", text: "This is my fourth post", comments_counter: nil, likes_counter: nil},
+  {author: User.find(2),title: "Hello", text: "This is my first post", comments_counter: nil, likes_counter: nil}
+  ]),
+  
+  Comment.create!([
+    {author_id: 1, post_id: 2, text: "commemt1"},
+    {author_id: 1, post_id: 2, text: "comment2"},
+    {author_id: 1, post_id: 2, text: "comment3"},
+    {author_id: 1, post_id: 2, text: "comment4"},
+    {author_id: 1, post_id: 2, text: "comment5"},
+    {author_id: 1, post_id: 2, text: "comment6"}
+  ])
```

Comment:
[OPTIONAL] After running `rails db:seed`, your code shows the following error:

<img width="647" alt="image" src="https://user-images.githubusercontent.com/77190203/174310426-6cdea07c-cd49-408d-8445-3ba3612ff9c4.png">

---

<img width="553" alt="image" src="https://user-images.githubusercontent.com/77190203/174311066-e1b910ad-83c1-402c-8239-8105d456733b.png">

 
It would be better if you fix those errors so that the users are able to seed the data in your app.

Reply:
Thanks @medaminedev66 👍🏼